test/e2e/page-objects/pages/home/bitcoin-homepage.ts

```diff
@@ -85,7 +85,7 @@ class BitcoinHomepage extends HomePage {
    */
   async check_isReceiveButtonEnabled(): Promise<boolean> {
     try {
-      await this.driver.findClickableElement(this.receiveButton, 1000);
+      await this.driver.findClickableElement(this.receiveButton);
```

Comment:
same

Reply:
Github messed it up when rebasing. Fixed